amundsen_application/static/js/components/TableDetail/index.tsx

```diff
@@ -204,6 +205,7 @@ export class TableDetail extends React.Component<
               <SourceLink tableSource={data.source} />
             </div>
             <div className="header-section header-buttons">
+              <TableReportsDropdown resourceReports={data.resource_reports} />
```

Comment:
nit: Can you add a test in the `render` test here to verify `TableReportsDropdown` is rendered and additionally could check if it is given the correct properties https://github.com/lyft/amundsenfrontendlibrary/blob/master/amundsen_application/static/js/components/TableDetail/index.spec.tsx#L66